NDVItimeseries.R

```diff
@@ -0,0 +1,527 @@
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Author: Riley Rudd
+# Project name: Sahel Adaptive Social Protection Program
+# Date Created: # Fri Jul 1 2022 ------------------------------
+# Date Last Updated: Tues Jul 5 2022
+# R version: 4.1.3
+# Purpose: Load and Clean NDVI Data Files
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Clear Environment and Set Options -----
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+rm(list = ls(all = TRUE))
+
+remove_chart_clutter <- 
+  theme(    
+    panel.grid.major = element_blank(),      # Remove panel grid lines
+    panel.grid.minor = element_blank(),      # Remove panel grid lines
+    panel.background = element_blank(),      # Remove panel background
+    axis.line = element_line(colour = "grey"),       # Add axis line
+    axis.title.y = element_text(angle = 0, vjust = 0.5),      # Rotate y axis so don't have to crank head
+    legend.position="bottom"
+  ) 
+
+# Don't show scientific notation
+options(scipen = 999)
+
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Load Libraries and Options -----
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+library(tidyverse)
```

Comment:
tidyverse includes several of the packages listed below, so don't need to call them explicitly. In particular, tidyverse includes the following:

[ggplot2](https://ggplot2.tidyverse.org/), for data visualisation.
[dplyr](https://dplyr.tidyverse.org/), for data manipulation.
[tidyr](https://tidyr.tidyverse.org/), for data tidying.
[readr](https://readr.tidyverse.org/), for data import.
[purrr](https://purrr.tidyverse.org/), for functional programming.
[tibble](https://tibble.tidyverse.org/), for tibbles, a modern re-imagining of data frames.
[stringr](https://github.com/tidyverse/stringr), for strings.
[forcats](https://github.com/tidyverse/forcats), for factors.

would recommend removing explicit calls of those packages